adonis-typings/redirect.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * @adonisjs/http-server
+ *
+ * (c) Harminder Virk <virk@adonisjs.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare module '@ioc:Adonis/Core/Redirect' {
```

Comment:
Why put it inside it's own module? I believe declaring the interface within the response module is fine?